artio-core/src/main/java/uk/co/real_logic/artio/protocol/GatewayPublication.java

```diff
@@ -125,6 +127,8 @@ public class GatewayPublication extends ClaimablePublication
     private static final int THROTTLE_CONFIGURATION_REPLY_LENGTH = HEADER_LENGTH +
         ThrottleConfigurationReplyEncoder.BLOCK_LENGTH;
     private static final int SEQ_INDEX_SYNC_LENGTH = HEADER_LENGTH + SeqIndexSyncEncoder.BLOCK_LENGTH;
+    private static final int LIBRARY_TIMEOUT_LENGTH = HEADER_LENGTH + LibraryTimeoutEncoder.BLOCK_LENGTH +
+        GroupSizeEncodingEncoder.ENCODED_LENGTH * 2;
```

Comment:
Why * 2?

Reply:
Sorry not needed for this one. Removed. Thanks.